api/Utils/Validation.cs

```diff
@@ -47,5 +68,17 @@ public bool checkDateFormat(string date)
                 return DateTime.TryParseExact(date, format, CultureInfo.InvariantCulture, DateTimeStyles.None, out datetime);
             }
         }
+
+        public bool checkLeaveDates(List<LeaveDateRequest> leaveDates)
+        {
+            if (leaveDates == null || leaveDates.Count == 0) return false;
+            return true;
```

Comment:
Try and check if this will work.

```suggestion
            return (leaveDates == null || leaveDates.Count == 0);
```